packages/build/src/plugins_core/functions/index.js

```diff
@@ -82,6 +85,21 @@ const zipFunctionsAndLogResults = async ({
   }
 }
 
+// `getFunctionPaths` will throw if the directory doesn't exist. This is the
+// expected behavior when dealing with the functions directory defined in the
+// config, but we don't want to throw if the internal functions directory does
+// not exist. To that effect, we wrap it with this try/catch and simply return
+// an empty array if `getFunctionPaths` fails.
+const listInternalFunctions = async (internalFunctionsSrc) => {
+  try {
+    return await getFunctionPaths(internalFunctionsSrc)
+  } catch (_) {
+    // no-op
+  }
+
+  return []
```

Comment:
[sand] Should this line be moved inside the `catch` block?

Reply:
Done in https://github.com/netlify/build/pull/3213/commits/db0334c9afccc05a5a009a09a7dcf29b9bb0e037!